### PR TITLE
Upload profile picture layout is distorted when the height of the picture is too large #2306

### LINKED ISSUE #2306: Upload profile picture layout is distorted when the height of the picture is too large
As title

### PR DESCRIPTION
**Pull request by thyageshm**
Fixes #2306

### CODE REVIEW

File: src/test/java/teammates/test/cases/ui/browsertests/StudentProfilePageUiTest.java [hunk 1]
```diff
@@ -141,11 +141,12 @@ private void testActions() throws Exception {
         profilePage.verifyStatus(Const.StatusMessages.STUDENT_PROFILE_PIC_TOO_LARGE);
         verifyPictureIsPresent(prevPictureKey);
         
-        ______TS("success case, update picture");
+        ______TS("success case, update picture (to long)");
```
Comment (damithc) [inline src/test/java/teammates/test/cases/ui/browsertests/StudentProfilePageUiTest.java]: to --> too ?

File: src/test/java/teammates/test/cases/ui/browsertests/StudentProfilePageUiTest.java [hunk 2]
```diff
@@ -141,11 +141,12 @@ private void testActions() throws Exception {
...
         
-        profilePage.fillProfilePic("src/test/resources/images/profile_pic_updated.png");
+        profilePage.fillProfilePic("src/test/resources/images/image_long.jpg");
```
Comment (damithc) [inline src/test/java/teammates/test/cases/ui/browsertests/StudentProfilePageUiTest.java]: long is a big ambiguous. tall?

**Comment by damithc** [comment]
Can merge after addressing the comments.

File: src/test/java/teammates/test/cases/ui/browsertests/StudentProfilePageUiTest.java (hunk 1 above)
Comment (thyageshm) [inline src/test/java/teammates/test/cases/ui/browsertests/StudentProfilePageUiTest.java]: hahaha sorry sir meant to say "updated picture to a long one". will update it :-)

**Pull request merged**